app/views/categories/show.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="flex gap-2 flex-col mb-4 ">
+  <h2 class="bg-sky-700/70 text-3xl flex text-slate-200 justify-center h-12">Transactions</h4>
+  <div class="flex justify-end">
+    <p class="bg-green-400 rounded-sm font-semibold" >Total Cost: $<%= @total_cost %>.00</p>
+  </div>
+  <% @expense.each_with_index do | x, index|%>
+    <%= link_to expense_path(x.id ) do %>
+      <div class=" flex flex-row h-max  border-0 gap-9 bg-white rounded-md  shadow-md shadow-slate-300 p-2 ">
+        <% @category.each  do | i|%>
+          <img src="<%= i.icon%>" class="h-28 w-28 rounded-sm mr-2" >
+        <%end%> 
+        <div class="flex flex-col">
+          <h2 class="text-xl font-semibold"><%= x.name%></h2>
+          <p> Date: <%= x.created_at.strftime('%d-%m-%Y') %> </p>
+          <p> Time: <%= x.created_at.strftime('%H:%M:%S') %> </p>
+        </div>
+        <p class="font-light">Cost: $<%= x.amount %>.00 </p>
+      </div>
+    <%end%>
+  <%end%>
+  <div class="flex gap-40">
+    <%= link_to "Add transaction ", new_expense_path,class:'border border-blue-300 shadow-sm shadow-sky-300 bg-blue-400 p-2 rounded-lg cursor-pointer hover:bg-blue-500 w-40 justify-center flex'%>
+    <%= link_to "Back" , categories_path, class:'border border-blue-300 shadow-sm shadow-sky-300 bg-blue-400 p-2 rounded-lg cursor-pointer hover:bg-blue-500 w-20 justify-center flex' %>
+  </div>
+</div>
```

Comment:
- [ ] I can't see the total amount of the transaction on the category page. Please add it. [Requirement](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=For%20each%20category%2C%20the%20user%20can%20see%20their%20name%2C%20icon%20and%20the%20total%20amount%20of%20all%20the%20transactions%20that%20belongs%20to%20that%20category.)

![image](https://user-images.githubusercontent.com/38635158/167254039-7d3eb7db-4016-42b6-bf06-9bcecc1c79a7.png)